math/perimeter.cpp

```diff
@@ -45,12 +45,13 @@ T rect_perimeter(T length, T width) {
  * @brief perimeter of a [triangle](https://en.wikipedia.org/wiki/Triangle) (a +
  * b + c)
  * @param base is the length of the bottom side of the triangle
- * @param height is the length of the tallest point in the triangle
+ * @param left_side is the length of the left side corresponding to the base of triangle
+ * @param right_side is the length of the right side corresponding to the base of triangle
  * @returns perimeter of the triangle
  */
 template <typename T>
-T triangle_perimeter(T base, T height, T hypotenuse) {
-    return base + height + hypotenuse;
+T triangle_perimeter(T base, T left_side, T right_side) {
```

Comment:
I prefer to see the "height & base" instead of "left and right" and the former is more intuitive.

Reply:
` return base + height + hypotenuse;` This was apt